src/app.rs

```diff
@@ -1165,7 +1176,12 @@ impl SearchState {
     }
 
     fn is_string_match(&self, s: &str) -> bool {
-        s.contains(&self.query)
+        match self.whole_word {
```

Comment:
Since it's just a `bool`, let's make this an `if` statement.

Reply:
https://github.com/StanfordLegion/prof-viewer/pull/32/commits/8db6655b410876567ad23fc5ab9c172f1ced41f5